add callout for single vs multiple API keys (#37)
mirainc/developer-docs: content/docs/300-on-brand-menu-api/300-api-keys.mdx

   ie. `a98b440c16414f9c8a8c3fc239ce84fe`

> <Callout>**Single API Key vs Many API Keys**: Although it's possible to create a unique API Key for each of your locations and menus, creating just a single API Key for all locations and menus simplifies API Key management and takes the guess-work out of remembering which key goes with which menus and locations.</Callout>

Reviewer: ```suggestion
<Callout>**Single vs multiple API Keys**: You can create a single API key for multiple locations and menus to avoid remembering which key goes with which menus and locations.</Callout>
```

> <Callout>**Single API Key vs Many API Keys: Although it's possible to create a unique API Key for each of your locations and menus, creating just a single API Key for all locations and menus simplifies API Key management and takes the guess-work out of remembering which key goes with which menus and locations.** </Callout>

Reviewer 2: I don't think we should bold the entire statement but rather just the beginning: 

```suggestion
<Callout>**Single API Key vs Many API Keys**: Although it's possible to create a unique API Key for each of your locations and menus, creating just a single API Key for all locations and menus simplifies API Key management and takes the guess-work out of remembering which key goes with which menus and locations.</Callout>
```

Reviewer 2: The callout is a bit big. What do you think about changing it to something like this:

**Single API Key vs Many API Keys**: You can create a single API key for multiple locations and menus to avoid remembering which key goes with which menus and locations.

<img width="658" alt="image" src="https://user-images.githubusercontent.com/1163252/180441197-b69cec2c-0fb1-450b-ae26-d6bd90325c49.png">

Author: @jschr agreed on bolding just the beginning. I was trying to be clear and knew it was getting a bit long, so was looking for suggestions, thanks for the update.

can you share the raw code so I can see what the font color and bold syntax looks like?

Reviewer 2: The `**` wraps the text you want to bold: 

```
<Callout>**Single vs multiple API Keys**: You can create a single API key for multiple locations and menus to avoid remembering which key goes with which menus and locations.</Callout>
```

Author: ah, just an oversight, I put the closing in the wrong place
